### PR TITLE
Qol items fix example

### PR DESCRIPTION
**Pull request by magnetikonline**
- Bumped to Golang 1.16
- Bumped `golangci-lint` version.
- Updated `README.md` example to use `go install` over `go get`.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,9 +1,15 @@
...
 	github.com/google/go-github/v33 v33.0.0
-	golang.org/x/oauth2 v0.0.0-20180821212333-d2e6202438be
+	github.com/google/go-querystring v1.1.0 // indirect
+	golang.org/x/crypto v0.0.0-20210322153248-0c34fe9e7dc2 // indirect
+	golang.org/x/mod v0.4.2 // indirect
+	golang.org/x/net v0.0.0-20210326060303-6b1517762897 // indirect
+	golang.org/x/oauth2 v0.0.0-20210323180902-22b0adad7558
 	golang.org/x/tools v0.1.0
+	google.golang.org/appengine v1.6.7 // indirect
 )
```
Comment (danielwhite) [inline go.mod:15]: Why did this list change so significantly if the main dependencies didn't!?

Comment (magnetikonline) [reply go.mod:15]: I'm thinking maybe an up to date `go.mod` wasn't in `main` before I started this branch. So I'm probably just brining it up to sync.

**Review by danielwhite — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (danielwhite) [reply go.mod:15]: Probably. Is it worth bumping the GitHub dependency while we're here?

Comment (magnetikonline) [reply go.mod:15]: Good call - bumped with adc6710.

**Pull request merged**